lib/Transforms/IPO/PassManagerBuilder.cpp

```diff
@@ -360,6 +360,7 @@ void PassManagerBuilder::populateModulePassManager(
     if (!HLSLHighLevel) {
       MPM.add(createDxilConvergentClearPass());
       MPM.add(createDxilRemoveDeadBlocksPass());
+      MPM.add(createDxilNoOptSimplifyInstructionsPass());
       MPM.add(createDxilNoOptLegalizePass());
```

Comment:
Did you mean to leave this earlier call to NoOptLegalizePass here in addition to the new one below?

Reply:
Oops, nice catch. Didn't impact result much though. Fixed.